src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -25,12 +24,12 @@
 import torch
 from torch import Tensor, nn
 
+from transformers import AutoBackbone, MaskFormerSwinConfig, SwinConfig
```

Comment:
Those should be relative imports pointing to their respective files (as it was for Swin before).